datacenter/ucp/2.1/guides/admin/configure/content-trust/admin_tasks.md

```diff
@@ -0,0 +1,387 @@
+    Click the **REQUIRE SIGNATURE FROM ALL OF THESE TEAMS** field and choose
+    one or more teams. DDC will consider an image to be trustworthy only if it
+    is signed by a member of _every team_ you select. The requirement can be
+    fulfilled by a single user who is a member of all the teams, or by a signer
+    in each team.
+
+    > **Note**: If you don't specify any team (by leaving the field blank), an
+    > image will be trusted as long as it is signed by any UCP user whose keys
+    > are [configured in the Notary client](#set-up-the-docker-notary-client).
+
+    The following screenshot shows a configuration that requires images to be
+    signed by a member of the `engineering` team.
+
+    ![Content Trust settings detail](/datacenter/images/ucp_content_trust_detail.png)
+
+3.  Click **Update** to apply the changes.
+
+UCP is now configured to only allow use of signed images, but you don't have the
+ability to sign images yet. Next,
+[set up the Docker Notary CLI client](#set-up-the-docker-notary-client).
+
+## Set up the Docker Notary CLI client
+
+After [configuring UCP](#configure-ucp), you need to specify which Docker images
+can be trusted, using the Docker Notary server that is built into Docker Trusted
+Registry (DTR). The following procedure configures the Notary server to store
+signed metadata about the Docker images you trust. This set-up step only needs
+to be done on the client of an administrator responsible for setting up
+repositories and delegating the ability to sign images.
+
+1.  **If you are on a Linux client**, install the Notary binary. If you use Docker
+    for Mac or Docker for Windows, the Notary client is included in your
+    installation.
+
+    - [Download the latest client archive](https://github.com/docker/notary/releases).
+    - Extract the archive.
+    - Rename the binary to `notary` and set it to executable. Either move it to
+      a location in your path or modify the examples below to include the full
+      path to the binary.
+
+2.  Configure the Notary client to communicate with the DTR server and store
+    its metadata in the correct location. You can either use a
+    [Notary configuration file](/notary/reference/client-config.md) or manually
+    specify the following flags when you run the `notary` command.
+
+    |Flag               |Purpose                                               |
+    |-------------------|------------------------------------------------------|
+    | `-s <dtr_url>`    | The hostname or IP address of the DTR server         |
+    | `-d <trust_directory>` | The path to the local directory where trust metadata will be stored |
+    | `--tlscacert <dtr_ca.pem>` | The path to the trust certificate for DTR. Only required if your DTR registry is not using certificates signed by a globally trusted certificate authority, such as self-signed certificates. Download the trust certificate from `https://<dtr_url>/ca` either from your browser or using `curl` or `wget`.|
```

Comment:
Non-blocking:  perhaps "CA certificate" as opposed to "trust certificate", just to distinguish it from the rest of the trust data when discussing notary?